modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorAriaLabelTest.ts

```diff
@@ -144,46 +116,72 @@ describe('browser.tinymce.themes.silver.editor.color.ColorAriaLabelTest', () =>
         await pResetColorToDefault(editor, selectors.backcolorSplitButton());
       });
 
-      it('TINY-9697: Forecolor - default color appended to aria-label', () => {
```

Comment:
It looks like this replaced all the tests for the icon section of the split button with chevron menu tests? Don't we need both?